linera-execution/src/evm/revm.rs

```diff
@@ -66,6 +66,10 @@ const GET_DEPLOYED_BYTECODE_SELECTOR: &[u8] = &[21, 34, 55, 89];
 /// The json serialization of a trivial vector.
 const JSON_EMPTY_VECTOR: &[u8] = &[91, 93];
 
+/// The maximum size of a deployed EVM contract
+/// Note: The limit of Etherem contract is 24576.
+const MAX_SIZE_EVM_CONTRACT: usize = 10000000;
```

Comment:
```suggestion
const MAX_SIZE_EVM_CONTRACT: usize = 10_000_000;
```

(Or `10 << 20`, if we want to make it 10 MiB?)